WooCommerce/Classes/ViewRelated/Products/Add Product/AddProductWithAI/StartingInfo/PromptProgressBar/ProductCreationAIBarProgressStyle.swift

```diff
@@ -0,0 +1,38 @@
+import SwiftUI
+
+struct ProductCreationAIBarProgressStyle: ProgressViewStyle {
+
+    var color: Color = .gray
+
+    func makeBody(configuration: Configuration) -> some View {
+
+        let progress = configuration.fractionCompleted ?? 0.0
+
+        GeometryReader { geometry in
+            VStack(alignment: .leading) {
```

Comment:
Do we need the `VStack`? I think it can be safely removed.

Reply:
8421de758d8b410109ab6a3675a9d562b0e1051c